ps2_setup.R

```diff
@@ -0,0 +1,221 @@
+# race: White, Asian-Pac-Islander, Amer-Indian-Eskimo, Other, Black.
+# sex: Female, Male.
+# capital-gain: continuous.
+# capital-loss: continuous.
+# hours-per-week: continuous.
+# native-country: United-States, Cambodia, England, Puerto-Rico, Canada, Germany, Outlying-US(Guam-USVI-etc), India, Japan, Greece, South, China, Cuba, Iran, Honduras, Philippines, Italy, Poland, Jamaica, Vietnam, Mexico, Portugal, Ireland, France, Dominican-Republic, Laos, Ecuador, Taiwan, Haiti, Columbia, Hungary, Guatemala, Nicaragua, Scotland, Thailand, Yugoslavia, El-Salvador, Trinadad&Tobago, Peru, Hong, Holand-Netherlands.
+# salary: >50K,<=50K
+
+# Further description is available in adult.names file
+
+# 1) read the data, make sure columns are read in the right format, i.e. character, double, etc.
+install.packages(c('jsonlite','readxl', 'writexl','purrr', 'foreign'))
+
+library(magrittr)
+library(tidyverse)
+library(purrr)
+library(foreign)
+setwd("C:/Users/Michal/Documents/GitHub/RfEX_stud_version/ps2")
+adult <- read.csv("~/GitHub/RfEX_stud_version/ps2/adult.data", header=FALSE)
+as.data.frame(adult)
+adult_names <- read.csv("~/GitHub/RfEX_stud_version/ps2/adult.names", header=FALSE)
+
+
+# 2) How many men and women are represented in the dataset?  32561W, 32560M
+num_female <- adult %>% select(V10)
+df <- data.frame(matrix(unlist(num_female), nrow=length(num_female), byrow=F))
+df <- t(df)
+df <- as.data.frame(df)
+
+num_female <- adult %>% select(V10) %>%
+  filter(V10 == " Female") %>%
+  nrow(.) %>%
+  print()
+ 
+
+num_male <- adult %>% select(V10) %>%
+  filter(V10 == " Male") %>%
+  nrow(.) %>%
+  print()
+
+# 3) What is the average age of women
+female_age <- adult %>% select(V1, V10) %>%
+  filter(V10 == " Female") %>%
+  na.omit() %>%
+  select(V1) %>%
+  summarise(mean(V1, na.rm = T)) %>%
+  print()
+
+# 4) What are the mean and standard deviation of age for those who earn more than 50K per year
+# and those who earn less than 50K per year
+mean_rich <- adult %>% select(V1, V15) %>%
+  filter(V15 == " >50K") %>%
+  select(V1) %>%
+  summarise(mean(V1, na.rm = T)) %>%
+  print()
+
+mean_poor <- adult %>% select(V1, V15) %>%
+  filter(V15 == " <=50K") %>%
+  select(V1) %>%
+  summarise(mean(V1, na.rm = T)) %>%
+  print()
+
+sd_rich <- adult %>% select(V1, V15) %>%
+  filter(V15 == " >50K") %>%
+  select(V1) %>%
+  summarise(sd(V1, na.rm = T)) %>%
+  print()
+
+sd_poor <- adult %>% select(V1, V15) %>%
+  filter(V15 == " <=50K") %>%
+  select(V1) %>%
+  summarise(sd(V1, na.rm = T)) %>%
+  print()
+
+# 5) Is it true that people who earn more than 50K have at least high school education? (education - 
+#     Bachelors, Prof-school, Assoc-acdm, Assoc-voc, Masters or Doctorate feature)
+education <- adult %>% select(V4, V15) %>%
+    filter(V15 == " >50K")
+
+table(education)
+sort(table(education),decreasing=TRUE)
+
+table(adult[which(adult$V15==" >50K")])
+#HOW TO SORT IN TABLE TO PRESERVE EDU NAMES?

```

Comment:
You  have to have either colnames or rownames set to educ levels to preserve them.

Anyway, there is no answer to the question
-5 points